test/integration/authenticated_user_cannot_act_as_a_admin_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class AuthenticatedUserCannotActAsAAdminTest < ActionDispatch::IntegrationTest
+  test "test's that an authenticated user cannot view admin data or change role" do
+    User.create(username:'Jhun', password:'dana', role: 0)
```

Comment:
Space missing after colon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.